validators_instructions.md

```diff
@@ -0,0 +1,139 @@
+
+Since Sommelier is a Cosmos chain, in order to provide strategies on Ethereum it needs a way to bridge assets and logic calls. Steward is responsible for running the Orchestrator, which handles relaying Cosmos transactions to Ethereum, and co-processing Ethereum transactions on Sommelier. Steward runs the Orchestrator so that Sommelier can manage Cellars on Ethereum.
+
+## Setting Up Steward
+
+In this section, let’s explore setting up steward for validators. First, ensure the sommelier chain is running. Next, create a `toml` file in the root of the application which will hold your configuration. To get a template for your configuration file, run the command below:
+
+```rust
+steward -- print-config
+```
+
+Replace the default keys in the template displayed in your terminal with your configuration. Now you have the configurations set up, let’s go through stewards commands.
+
+## Start Cosmos Mode
+
+To start the Allocation signer mode, ensure that the server section in your config file is set properly as shown below. The `address`, `client_ca_cert_path` and `port` are optional fields.
+
+```rust
+keystore = "/tmp/keystore"
+
+[server]
+address = "127.0.0.1"           // This is an optional feild.
+client_ca_cert_path = ""       // optional, defaults to VolumeFi client cert
+port = 9999                   // optional, default is 5734
+server_cert_path = ""
+server_key_path = ""
+
+[gravity]
+contract = "0x0000000000000000000000000000000000000000"
+fees_denom = "stake"
+
+[ethereum]
+key_derivation_path = "m/44'/60'/0'/0/0"
+rpc = "http://localhost:8545"
+gas_price_multiplier = 1.0
+blocks_to_search = 5000
+
+[cosmos]
+key_derivation_path = "m/44'/118'/0'/0/0"
+grpc = "http://localhost:9090"
+prefix = "cosmos"
+msg_batch_size = 5
+gas_adjustment = 1.0
+
+[cosmos.gas_price]
+amount = 0.001
+denom = "stake"
+
+[metrics]
+listen_addr = "127.0.0.1:3000"
+
+[[cellars]]
+pair_id = "0x0"
+name = ""
+token_id = "0x0"
+cellar_address = "0x0000000000000000000000000000000000000000"
+pool_address = "0x0000000000000000000000000000000000000000"
+weight_factor = 100
+max_gas_price_gwei = 100
+
+[cellars.token_0]
+decimals = 18
+symbol = "NA"
+address = "0x0000000000000000000000000000000000000000"
+
+[cellars.token_1]
+decimals = 18
+symbol = "NA"
+address = "0x0000000000000000000000000000000000000000"
+
+[cellars.duration]
+secs = 60
+nanos = 0
+
+[keys]
+keystore = "/tmp/keystore"
+rebalancer_key = ""
+```
+
+Next, run the cosmos signer command to start the cosmos mode:
+
+```rust
+steward -- -c [your_config_file_name.toml] cosmos-signer
+```
+
+## Start Orchestrator
+
+Steward allows you to start the Orchestrator with or without the Relayer. First, you’ll need an Ethereum key and a Cosmos key. Run the command below to create your keys if you don’t have one.
+
```

Comment:
Let's also include an example here of how to import existing keys.

Reply:
fixed